pkg/archive/archive.go

```diff
@@ -319,6 +319,33 @@ func WriteZipToTar(tw TarWriter, srcZip, basePath string, uid, gid int, mode int
 	return nil
 }
 
+func WriteFileToTar(tw TarWriter, srcFile, destPath string, uid, gid int, mode int64, normalizeModTime bool) error {
```

Comment:
Is it possible to write a test for this?

Reply:
done!